src/ucp/core/ucp_request.inl

```diff
@@ -164,6 +164,50 @@ void ucp_request_recv_generic_dt_finish(ucp_request_t *req)
     }
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_request_send_state_set(ucp_request_t *req, size_t data_offset,
```

Comment:
Is this for future use?

Reply:
Yes... will move this and below to next PRs